bundles/org.openhab.binding.exec/src/main/java/org/openhab/binding/exec/internal/handler/ExecHandler.java

```diff
@@ -97,9 +109,8 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                     lastInput = command.toString();
                     if (lastInput != null && !lastInput.equals(previousInput)) {
                         if (getConfig().get(AUTORUN) != null && ((Boolean) getConfig().get(AUTORUN)).booleanValue()) {
-                            lastInput = command.toString();
```

Comment:
Don't these changes break the way `lastInput` is used?

Reply:
Here I just removed the duplicate of this `lastInput = command.toString();`.
If you mean as a whole, no. lastInput can still be used as multiple arguments if no quotes are used (in fact it's exactly the same, just that using it as one argument surrounded by quotes now works)